engine/runtime/src/main/java/org/enso/interpreter/runtime/builtin/Builtins.java

```diff
@@ -289,7 +391,12 @@ private Map<Class<? extends Builtin>, Builtin> initializeBuiltinTypes(
             throw new CompilerError("Invalid builtin type entry: " + constr);
           }
         });
-    builtins.values().forEach(b -> b.initialize(language, scope, builtins));
+    builtins
+        .values()
+        .forEach(
```

Comment:
```
for (var b : builtins.values()) {
   // might be more effective
}
```
as it doesn't require any lambdas.

Reply:
But.. I love lambdas :)

Reply:
I reluctantly changed it to a for loop. I feel very old now.